src/trulens_eval/pyproject.toml

```diff
@@ -27,18 +27,18 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9,!=3.9.7"
+python = "^3.8.1,!=3.9.7"
 # If any of these below has a new release, make a new trulens_eval release with
 # the latest trulens-* versions for users who are still installing
 # "trulens_eval".
-trulens = { version = "^1.0" }
-#trulens-core = { version = "^1.0.0" }
-#trulens-feedback = { version = "^1.0.0" }
-#trulens-dashboard = { version = "^1.0.0" }
+# trulens = { version = "^1.0" }
+trulens-core = { version = "^1.0.0" }
```

Comment:
not sure why but swapping trulens -> trulens-core,-dashboard,-feedback seemed to fix some poetry resolution error.